pkg/injector/envoy_config.go

```diff
@@ -1,21 +1,18 @@
 package injector
 
 import (
-	"context"
 	"encoding/base64"
 	"fmt"
 	"strconv"
 
+	ghyaml "github.com/ghodss/yaml"
```

Comment:
Any reason to alias this import? Would drop the alias if not required.

Reply:
Yes - `yaml` is already taken in this context